### PR TITLE
update index.js

### PR DESCRIPTION
**Pull request by uniyalprashant9**
I merged my code with existing index.js .

But existing  index.js didn't worked for me
image

### CODE REVIEW

**Comment by uniyalprashant9** [comment]
OMG!!... It worked here but it was showing error in my laptop.

**Comment by atherdon** [comment]
ok, then merging. you can move to a next things

File: generator/writeFile.js [hunk 1]
```diff
@@ -51,7 +51,42 @@ function test () {
...
 
+
+
+
+/*
+This module will create a directory with the name of object file 
+and puts all the splitted elements inside the directory.
+*/
+const fs = require('fs');
+function splitObject() {
```
[the remark this answers is not in the document]
Comment (atherdon) [reply generator/writeFile.js]: @uniyalprashant9 check this review

**Comment by atherdon** [comment]
i don't think that removing that file is a good idea. you did a same move twice - code was written by someone and then i must be needed

**Comment by uniyalprashant9** [comment]
ok

**Comment by uniyalprashant9** [comment]
I will open a new pull request for that.

**Comment by atherdon** [comment]
@uniyalprashant9 I assume you need no learn how to sync your forked repo. because you deleting it and reforking it again. am I right?

**Comment by uniyalprashant9** [comment]
yes, because sometimes I mess up with some files in my fork. I will take care of that from now.

**Comment by atherdon** [comment]
no problem here - it's just a good practial way to improve your git skills

**Comment by atherdon** [comment]
this will help you with forking. btw, that repo is full of information about git - review it
https://github.com/ChickenKyiv/awesome-git-article#keeping-a-fork-up-to-date

**Comment by atherdon** [comment]
and don't forget to open a new pull request with that changes that you've made

**Pull request closed without merging**